packages/valory/skills/decision_maker_abci/behaviours/decision_request.py

```diff
@@ -71,6 +72,21 @@ def setup(self) -> None:
         msg = f"Prepared metadata {self.metadata!r} for the request."
         self.context.logger.info(msg)
 
+    def initialize_bet_id_row_manager(self) -> Dict[str, List[int]]:
+        bets_mapping = {}
+        dataset_filepath = (
+            self.params.store_path / self.benchmarking_mode.dataset_filename
+        )
+
+        with open(dataset_filepath, mode="r") as file:
+            reader = csv.DictReader(file)
+            for row_number, row in enumerate(reader, start=1):
+                question_id = row["question_id"]
```

Comment:
Why not use the `question_id_field` parameter?

Reply:
done